strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/security/ldap/LdapAuthenticatorConfigurationController.java

```diff
@@ -64,8 +184,8 @@ public ResponseEntity getRolesMapping(@RequestHeader(HttpHeaders.ACCEPT) String
                             @ApiResponse(code = 400, message = "LDAP is not enabled or LDAP role mapping already exists for given LDAP role"),
                             @ApiResponse(code = 500, message = "Failed to add LDAP role mapping") })
     @PostMapping(value = "/rolesMapping/{externalRole}/{internalRole}",
-                 produces = { MediaType.TEXT_PLAIN_VALUE,
-                              MediaType.APPLICATION_JSON_VALUE })
+            produces = { MediaType.TEXT_PLAIN_VALUE,
+                         MediaType.APPLICATION_JSON_VALUE })
     public ResponseEntity addRoleMapping(@PathVariable String externalRole,
```

Comment:
@fuss86 our role mappings stored only in memory? I mean that we have `AuthoritiesExternalToInternalMapper.rolesMapping` which is seems not persisted.

Reply:
This Controller's method updates `AuthoritiesExternalToInternalMapper`  so I'm not sure if I understand the problem, @sbespalov

Reply:
Cosmetic: I think this class should actually be named `ExternalToInternalAuthoritiesMapper`?

Reply:
@carlspring do you confirn ?:)